### ISSUE TITLE
Ability to have all pdfs in one build

### ISSUE THREAD

**Issue by kaydin-nguyen** [BrowserStackCE/percy-pdf#7]
Currently there is a different build for each document and it is cumbersome to approve each build. It would be nice to have one build per test run where all the documents are there for ease of approval. In addition, sorting pages per documents in numerical order would be a plus.

**Comment by samirans89** [comment]
Thank you @kaydin-nguyen for your feedback.

> Currently there is a different build for each document and it is cumbersome to approve each build. It would be nice to have one build per test run where all the documents are there for ease of approval. 

This feature request is being considered as a part of a bigger change, in our attempt to simplify the PDF Run Info Config file further. You can refer the Pull Request: [Multiple Config file enhancements + Win support + Configurable port #6](https://github.com/BrowserStackCE/percy-pdf/pull/6) for updates and in case you are interested to try out the nightly incremental updates and pass on the feedback before it is merged to `main`.

> In addition, sorting pages per documents in numerical order would be a plus.

Currently, this is in reverse order on the Percy dashboard by default. There is a search functionality on the Percy dashboard, once you get inside the Percy Build. (you can find the search visual icon on top left of the screen). Once you search based on the file name, the results are in order. i.e. Page 1, Page 2 and so on etc. Would that be helpful? Once we have multiple PDFs in a single build, I think that would be the "go to" option for user too, since these PDF snapshots are taken in parallel by the `percy snapshot` command and hence maintaining specific ordering could be tricky in a single build.
